specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -9134,8 +9272,99 @@
         }
       }
     },
+    "VaultHealthDetails": {
+      "description": "Vault health details definition.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/VaultHealthProperties",
+          "description": "The vault health related data."
+        }
+      }
+    },
+    "VaultHealthProperties": {
+      "description": "class to define the health summary of the Vault.",
+      "type": "object",
+      "properties": {
+        "vaultErrors": {
+          "description": "The list of errors on the vault.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthErrorSummary"
+          }
+        },
+        "protectedItemsHealth": {
+          "$ref": "#/definitions/ResourceHealthSummary",
+          "description": "The list of the health detail of the protected items in the vault."
+        },
+        "fabricsHealth": {
+          "$ref": "#/definitions/ResourceHealthSummary",
+          "description": "The list of the health detail of the fabrics in the vault."
+        }
+      }
+    },
+    "HealthErrorSummary": {
+      "description": "class to define the summary of the health error details.",
+      "type": "object",
+      "properties": {
+        "summaryCode": {
```

Comment:
ok, thanks, yes, I would recommend doing that in your next version if you don't see it being possible now (or causes a breaking change).